documentation/architecture/README.md

```diff
@@ -1,29 +1,60 @@
+terminology as they produce (Spill) data to the Engine. This terminology is independent of
+Kafka producer/consumer terminology. In fact the producer threads are Kafka consumers.
 
-There is one thread per topic (worker_run in ESSStream.cpp). The exact number of threads depends on the project configuration. There are other producers, but for most (all?) detector data we use the ESSStream producer class.
+There is one thread per topic (**worker_run()** in *ESSStream.cpp*). The exact number of
+threads depends on the project configuration. There are other types of producers, but for
+most (all?) detector data we use the **ESSStream** producer class.
 
-![screenshot](images/2020_06_data_path.png)
+![screenshot](images/data_path.png)
 
-Producers are passed a pointer to an output queue object, so they could write to different queues. In practice (as of 2020-06) they all write to the same queue (parsed_queue).
+Producers are passed a pointer to an output queue object, so they could write to different
+queues. In practice (as of 2020-06) they all write to the same queue (**parsed_queue**).
 
-Another thread (builder_naive in Engine.cpp) continuoisly pull 'Spills' from 'parsed_queue' and pass them on (copying/duplicating?) to individual consumer queues.
+Another thread (**builder_naive** in *Engine.cpp*) continuously pull **Spills** from
+**parsed_queue** and pass them on (copying/duplicating?) to the Consumers' individual
+consumer queues.
 
 ### Daquiri consumers
-Daquiri has a numbers of consumers such as 'Histogram3D', 'Histogram2D', 'TOF1D', etc. (search for 'public Spectrum' to identify consumers).
+Daquiri has a numbers of consumers such as **Histogram3D**, **Histogram2D**, **TOF1D**,
+etc. (search for 'public Spectrum' to identify consumers).
 
-Not all consumers are interested in all spills. Spills for individual consumers are accepted by the 'accpet_spill()' method. Furthermore not all events in a spill may be relevant. Events are accepted by the 'accept_event()' method.
+Not all consumers are interested in all spills. Spills for individual consumers are accepted
+by the **accept_spill()** method. Furthermore not all events in a spill may be relevant.
+Events are accepted by the **accept_event()** method.
 
 
 ## GUI
-To be created - Not available yet
+This is mostly a description on the workflow from startup to running an acquisition. Other
+relevant topics such as creating and editing profiles and plots are currently omitted.
+
+### Starting Daquiri
+This workflow assumed that Daquiri starts from a state where a profile is loaded but has not
+been started (i.e. the Boot button is shown/active).
+
+#### Boot from current settings
+This is the boot sequence when a (detector) profile has already been loaded.
+![screenshot](images/gui_boot_from_settings.png)
+
+#### Boot from settings dialog
+This is the boot sequence when a new (detector) profile has been chosen from the Dialog.
+![screenshot](images/gui_boot_from_profiles.png)
```

Comment:
The **ProfilesForm** box has a typo (_Qi_ instead of _Qt_).

Reply:
will fix